packages/stark-core/src/modules/session/services/session.service.spec.ts

```diff
@@ -236,6 +236,7 @@ describe("Service: StarkSessionService", () => {
 
 			expect(hookMatchCriteria.entering).toBeDefined();
 
+			/* tslint:disable-next-line:no-useless-cast no-unnecessary-type-assertion */
 			const matchingFn: Predicate<StateObject> = <Predicate<StateObject>>hookMatchCriteria.entering;
```

Comment:
Since we're casting the type here, could we remove the type definition ? :blush: 
`const matchingFn = <Predicate<StateObject>>hookMatchCriteria.entering;`